src/screens/Product/index.jsx

```diff
@@ -81,23 +81,24 @@ const Product = React.memo(() => {
     }
   };
 
-  const addToBag = async () => {
+  const addToCart = async () => {
     if (!selectedSize) {
       notify("Please select your desired size.", {
         type: "warning"
       });
     } else {
       try {
-        await addItemToBag({
+        addItemToCart({
```

Comment:
missing `await` before method I think

Reply:
it doesn't matter, since it doesn't resolve to anything